xdsl/utils/hints.py

```diff
@@ -17,10 +18,13 @@
 from xdsl.ir import ParametrizedAttribute
 from xdsl.utils.exceptions import VerifyException
 
+if TYPE_CHECKING:
+    from typing_extensions import TypeForm
+
 _T = TypeVar("_T")
 
 
-def isa(arg: Any, hint: type[_T]) -> TypeGuard[_T]:
+def isa(arg: Any, hint: "TypeForm[_T]") -> TypeGuard[_T]:
```

Comment:
What error do you get here?

Reply:
TypeForm is defined in Pyright, not typing_extensions, so the error is:
```
E   NameError: name 'TypeForm' is not defined
```